engines/bastion_katello/lib/bastion_katello/engine.rb

```diff
@@ -33,6 +33,7 @@ class Engine < ::Rails::Engine
         ),
         :config => {
           'consumerCertRPM' => consumer_cert_rpm,
+          'enable_download_policies' => !Foreman.in_rake? && Setting[:enable_download_policies],
```

Comment:
Hmm, it's still failing which is odd.  Right below this is another example (remoteExecutionByDefault) that only excludes it from db:migrate, so maybe try `!Foreman.in_rake?('db:migrate')` instead